bindings/wasm/tsconfig.json

```diff
@@ -8,6 +8,7 @@
         "importsNotUsedAsValues": "error",
         "noImplicitAny": true,
         "preserveConstEnums": true,
+        "esModuleInterop": true,
```

Comment:
This can be avoided by passing `--project tsconfig.node.json` to `ts-node` in the `example:node` script:

```
    "example:node": "ts-node --project tsconfig.node.json ./examples/src/main.ts",
```

I don't know if enabling `esModuleInterop` has unintended consequences when compiling for the web package, there are issues regarding the flag but I don't know if they are specific to browsers or not. @abdulmth , @eike-hass ?